### DISCUSSION TITLE
I want to change color of Vaultwarden Web

### DISCUSSION THREAD

**Discussion by GaetanJeff** [dani-garcia/vaultwarden#4807] [Q&A] [answered]
I want to change the Vaultwarden logo and color but I can't see the file to change it

**Comment by BlackDex** [comment]
See https://github.com/dani-garcia/bw_web_builds
You need to build the web-vault your self and use that.

**Reply by GaetanJeff** [reply]
In which file is the color of vaultwarden web-vault stored ?

**Reply by BlackDex** [reply, marked as answer]
Search the patch file for colors
